### PR TITLE
[DMS] Update `instance` pkg

### PR DESCRIPTION
**Pull request by lego963**
### What this PR does / why we need it
Update `instance` pkg

### Special notes for your reviewer

```
=== RUN   TestDmsList
--- PASS: TestDmsList (1.88s)
=== RUN   TestDmsLifeCycle
    instances_test.go:49: Attempting to create DMSv1 instance
    instances_test.go:86: DMSv1 instance successfully created: 0c45e0af-10db-4474-a644-9c72202f0030
    instances_test.go:103: Attempting to update DMSv1 instance: 0c45e0af-10db-4474-a644-9c72202f0030
    instances_test.go:113: DMSv1 instance updated successfully: 0c45e0af-10db-4474-a644-9c72202f0030
    instances_test.go:92: Attempting to delete DMSv1 instance: 0c45e0af-10db-4474-a644-9c72202f0030
    instances_test.go:99: DMSv1 instance deleted successfully: 0c45e0af-10db-4474-a644-9c72202f0030
--- PASS: TestDmsLifeCycle (371.59s)
PASS

Process finished with the exit code 0
```

### CODE REVIEW

File: acceptance/openstack/dms/v1/instances_test.go [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	storage, _ := strconv.Atoi(details.Storage)
+
+	createOpts := instances.CreateOpts{
+		Name:            dmsName,
+		Description:     "some interesting description",
+		Engine:          "kafka",
+		EngineVersion:   "2.3.0",
+		StorageSpace:    storage,
+		Password:        "5ecuredPa55w0rd!",
+		AccessUser:      "rgyrbu",
```
Comment (outcatcher) [inline acceptance/openstack/dms/v1/instances_test.go]: Make `AccessUser` less personate, e.g. `root`

File: openstack/dms/v1/instances/requests.go [hunk 2]
```diff
@@ -206,7 +217,7 @@ func List(client *golangsdk.ServiceClient, opts ListDmsBuilder) pagination.Pager
 		return DmsPage{pagination.SinglePageBase(r)}
 	})
 
-	dmsheader := map[string]string{"Content-Type": "application/json"}
-	pageDmsList.Headers = dmsheader
+	dmsHeader := map[string]string{"Content-Type": "application/json"}
+	pageDmsList.Headers = dmsHeader
```
Comment (outcatcher) [inline openstack/dms/v1/instances/requests.go]: Why `dmsHeader` var is needed?

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: openstack/dms/v1/instances/requests.go (hunk 2 above)
Comment (lego963) [reply openstack/dms/v1/instances/requests.go]: done

File: acceptance/openstack/dms/v1/instances_test.go (hunk 1 above)
Comment (lego963) [reply acceptance/openstack/dms/v1/instances_test.go]: done

**Review by outcatcher — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**